packages/block-library/src/container/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { InnerBlocks, getColorClassName } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export const name = 'core/container';
+
+export const settings = {
+	title: __( 'Container' ),
+
+	icon: <SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><Path d="M19 12h-2v3h-3v2h5v-5zM7 9h3V7H5v5h2V9zm14-6H3c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h18c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16.01H3V4.99h18v14.02z" /><Path d="M0 0h24v24H0z" fill="none" /></SVG>,
+
+	category: 'layout',
+
+	description: __( 'A container for other blocks.' ),
+
+	keywords: [ __( 'container' ), __( 'section' ), __( 'wrapper' ) ],
+
+	supports: {
+		align: [ 'wide', 'full' ],
+		anchor: true,
+		html: false,
+		reusable: false,
```

Comment:
I'm not sure I understand why this is not allowed as a reusable block?

Reply:
@youknowriad I don't understand this either. I think @talldan added so unless he can advise why it's been made thus I think we make it reusable again.

Reply:
Hmm, I can't remember why it was added... Seems like it can be removed.

Reply:
Done